### ISSUE TITLE
v3: Button does not switch when ALIAS is used

### ISSUE THREAD

**Issue by RkcCorian** [Zefau/ioBroker.jarvis#1035]
*Please note that you may restore an old layout via jarvis adapter configuration directly within ioBroker

**Prerequisites / Voraussetzungen**
- [x] Which jarvis version are you running / Welche Version von jarvis nutzt du? **BITTE GENAUE ANGABE!**
- [x] Fehler / Screenshot der Browser Konsole (F12), siehe [Fehleranalyse und -Meldung](https://github.com/Zefau/ioBroker.jarvis/wiki/de-Errors)

v3.0.0-alpha.80

**Describe the bug / Beschreibe den Fehler**
A clear and concise description of what the bug / problem is.
Eine genaue Beschreibung des Fehlers inkl. Herleitung, um diesen Nachstellen zu können.

Ich nutze einen boolean Datenpunkt. Diesen würde ich gerne mit einem Button steuern. (Play/Pause im Wechsel).
Dazu habe ich testhalber einen boolean Datenpunkt unter 0_userdata angelegt. Diesen dann mit einem ButtonAction gesteuert. Das funktioniert, sodass der Wert beim Button klicken von false > true > false wechselt, auch die Beschreibung wechselt mit.
Nun habe ich diesem DP hinter einen ALIAS gesetzt. Jetzt kann ich nur noch von false > true wechseln, jedoch nicht mehr auf false zurück.
Mir ist nicht klar was das mit dem ALIAS zu tun hat :-|

```
{
  "common": {
    "name": "Test",
    "desc": "Manuell erzeugt",
    "role": "state",
    "type": "boolean",
    "read": true,
    "write": true,
    "def": false
  },
  "type": "state",
  "from": "system.adapter.admin.0",
  "user": "system.user.admin",
  "ts": 1632604987554,
  "_id": "0_userdata.0.Wohnzimmer.Wohnzimmerlautsprecher.Test",
  "acl": {
    "object": 1636,
    "state": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  }
}
```
```
{
  "common": {
    "name": "Test",
    "desc": "Manuell erzeugt",
    "role": "state",
    "type": "boolean",
    "read": true,
    "write": true,
    "def": false,
    "alias": {
      "id": "0_userdata.0.Wohnzimmer.Wohnzimmerlautsprecher.Test"
    }
  },
  "type": "state",
  "_id": "alias.0.Wohnzimmer.Alexa_Wohnzimmer.Test",
  "acl": {
    "object": 1636,
    "state": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  },
  "from": "system.adapter.admin.0",
  "user": "system.user.admin",
  "ts": 1632605229861
}
```

**Screenshots**
Sofern möglich, bitte Screenshots einfügen, um den Fehler bildlich zu erläutern.
![image](https://user-images.githubusercontent.com/79741077/134787014-34f8eee1-ded0-4a1c-960b-cc86e9b7771b.png)
![image](https://user-images.githubusercontent.com/79741077/134787017-98beac60-e2c2-4d00-885c-7f975d241c2c.png)

____

**Please note that bug reports may be closed if the information above are not given!**

**Bitte beachte, dass Bug-Reports ggf. einfach geschlossen werden, wenn die oben genannten Kriterien nicht angemessen gegeben sind!**

**Comment by RkcCorian** [comment]
@Zefau: FYI bzgl. Performance und ALIAS: https://forum.iobroker.net/topic/25772/fragen-antworten-rund-um-die-neue-alias-funktion/386
Wo können wir das Problem mit ALIAS und dem BlindLevelAction verfolgen? Keine Ahnung ob das auch irgendwie mit Performance zu tun hat, ggf. eine schärfere Timeouterkennung?

**Comment by Zefau** [comment]
Timeout ist schon hochgestellt. Wüsste nicht, was ich noch in jarvis machen kann. 

Ich überlege, ob Jarvis nicht der device manager sein kann, statt der devices Adapter?

Man müsste die jarvis Geräte nur im blockly/Javascript nutzbar haben?

**Comment by RkcCorian** [comment]
Hmm... dafür kenne ich die ioBroker Interna zu wenig. Damit würdest Du ggf. in Konkurrenz / Redundanz zur Entwicklung der ALIAS-Entwicklung stehen, aber tlw. dasselbe machen?

**Comment by Zefau** [comment]
Ja, aber dadurch dass jarvis einen Import erlaubt aus meiner Sicht die bessere Alternative.

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/1147

**Comment by RkcCorian** [comment]
Verstehe... techn. für mich als Haupt-JARVIS-Nutzer besser. Ansonsten nutze ich für die verbleibenden 10-15% iFrames mit Grafana und VIS.

**Comment by RkcCorian** [comment]
Danke in jedem Fall! Jetzt warte ich gerade noch darauf, dass die letzten Bugs behoben werden, dann habe ich meine Migration auf v3 aber auch Erweiterung mit neuen Features abgeschlossen 😀
